CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Changelog
 ### next
+  * tpm2_sign: add option -D  to provide an already computed digest to sign
```

Comment:
Usually @williamcroberts updates the Changelog after things are merged. I don't know if he prefers to do it or is just that people don't usually update it.

I'm not against your change of course, I just don't know what is expected and if he has a preference (we should probably document it in CONTRIBUTING.md so people if they should update it or not).

Reply:
I was not sure if this change was a good idea. I've remove it. 

Thanks for your review!